.github/ISSUE_TEMPLATE/breaking_change.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Breaking Change
+about: Track plans for a breaking change
+title: "Removal of ? in v2."
+labels: "breaking change"
+assignees: ""
+---
+
+## Broken API or pattern
+
+<!-- Name the APIs involved specifically, etc. -->
+
+**Preparation References**
+
+<!-- A URL to the deprecation release notes. Alternatively, describe actions required where broken API or pattern is in use. -->
+
+**Packages**
+
+<!-- A simple list of impacted packages -->
+
+**Expected Timeline**
```

Comment:
This is implicit?

Reply:
This is probably the most annoying block for us to fill out. But should be good for clarity to reader. Might come to issue directly and not see the parent issue (or one wasn't linked) or maybe title doesn't have it. But never implicit.